src/github.rs

```diff
@@ -5,8 +5,8 @@ use super::USERAGENT;
 use config::Config;
 
 pub fn release(config: &Config, tag_name: &str, tag_message: &str) -> Result<(), Error> {
-    let user      = &config.user[..];
-    let repo_name = &config.repository_name[..];
+    let user      = &config.user.as_ref().unwrap()[..];
+    let repo_name = &config.repository_name.as_ref().unwrap()[..];
```

Comment:
I tried to resolve this one, but couldn't figure out how. Can you have a look at this?